demo/SliderDemo.qml

```diff
@@ -0,0 +1,119 @@
+import QtQuick 2.0
+import QtQuick.Layouts 1.1
+import Material 0.1
+
+Item {
+
+    Flickable {
+        id: flickable
+        clip: true
+        anchors.fill: parent
+        contentWidth: width
+        contentHeight: contents.height
+        interactive: contents.height > height
+
+        Item {
+            width: flickable.width
+            height:  childrenRect.height
+            id: contents
+
+            Column {
+                anchors.centerIn: parent
+                spacing: units.dp(20)
+
+                Repeater {
+                    model: 2
+
+                    Rectangle {
+                        color: index == 0 ? "#EEE" : "#333"
+                        width: grid.implicitWidth + units.dp(50)
+                        height: grid.implicitHeight + units.dp(50)
+                        radius: units.dp(2)
+
+                        GridLayout {
+                            id: grid
+                            anchors.centerIn: parent
+                            columns: 2
+                            rowSpacing: units.dp(20)
+                            columnSpacing: units.dp(10)
+
+                            Label {
+                                text: "Normal"
+                                color: index == 0 ? Theme.light.textColor : Theme.dark.textColor
+                            }
+
+                            Slider {
+                                Layout.alignment: Qt.AlignVCenter | Qt.AlignHCenter
+                                darkBackground: index == 1
+                                orientation: Qt.Vertical
```

Comment:
I created a new pull request that removes this, since this was already merged

https://github.com/papyros/qml-material/pull/80